study.md

```diff
@@ -134,15 +134,19 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, give a brief description of what HTTP is.
 
 ```md
-<!-- your answer here -->
+HTTP is a protocol which decides how documents are transferred.  Effectively it is a language.
+
+Above and provided readings.
 ```
 
 ## Describe what a client is and what a server is
 
  What is a client and what is a server? How do they interact with each other?
 
 ```md
-<!-- your answer here -->
+A client is is a centralized, powerful computer is a hub of which a less powerful, personal computer makes reuests to.
```

Comment:
I would more say that a client is an interface that a user interacts with, and that a server is what delivers the resources to that client's interface.